lib/rules/require-shorthand-arrow-functions.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Require shorthand arrow function expressions when returning a single statement.
+ *
+ * Type: `Boolean`
+ *
+ * Value: `true`
+ *
+ * Version: `ES6`
+ *
+ * #### Example
+ *
+ * ```js
+ * "requireShorthandArrowFunctions": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * evens.map(v => v + 1);
+ * evens.map(v => {
+ *     v = v + 1;
+ *     return v;
+ * });
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * evens.map(v => { return v + 1; });
```

Comment:
Well what I'm thinking is:
one form is `v => v + 1`.
The other is `{ return v => v + 1 }`.
The shorter form is only allowed when you return a single statement/expression - it becomes a implicit return.